libs/shared/db/firestore/jest.config.ts

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable */
```

Comment:
Q1: Does this need a copyright header.
Q2: Why disable ESLint for the file?

Reply:
Q1: Updated
Q2: Nx generates it this way. I assume it doesn't conform to ESLint rules, but not sure. We're following this generator for all source, so it's this way in every new lib.